src/services/microfrontends-manager.ts

```diff
@@ -0,0 +1,47 @@
+import * as path from 'node:path'
+import * as fs from 'node:fs'
+import { CLIError } from '@oclif/errors'
+import { BundleDescriptor, MicroFrontend } from '../models/bundle-descriptor'
+import BundleDescriptorManager from './bundle-descriptor-manager'
+
+const MICROFRONTENDS_DIRNAME = 'microfrontends'
+
+export default class MicroFrontendsManager {
```

Comment:
I suggest you, in general, avoid export default if favor of named export, on the scale can be problematic for [several reasons](https://basarat.gitbook.io/typescript/main-1/defaultisbad). If you use named export can take the advance of better navigation from the IDE, Typo protection, and class name protection. @cecchisandrone